morse_code.rb

```diff
@@ -0,0 +1,31 @@
+DICTINARY = {
+  'A' => '.-', 'B' => '-...', 'C' => '-.-.', 'D' => '-..', 'E' => '.', 'F' => '..-.',
+  'G' => '--.', 'H' => '....', 'I' => '..', 'J' => '.---', 'K' => '-.-', 'L' => '.-..',
+  'M' => '--', 'N' => '-.', 'O' => '---', 'P' => '.--.', 'Q' => '--.-', 'R' => '.-.',
+  'S' => '...', 'T' => '-', 'U' => '..-', 'V' => '...-', 'W' => '.--', 'X' => '-..-',
+  'Y' => '-.--', 'Z' => '--..', '0' => '-----', '1' => '.----', '2' => '..---',
+  '3' => '...--', '4' => '....-', '5' => '.....', '6' => '-....', '7' => '--...',
+  '8' => '---..', '9' => '----.'
+}.freeze
+
+def decode_char(letter)
+  DICTINARY.key(letter)
+end
+
+def decode_word(word_char)
+  word = ''
+  new_word = word_char.split
+  new_word.map { |char| word += char.split.map { |cha| decode_char(cha) }.join }
+  word
+end
+
+def decode(str)
+  new_str = ''
+  new_word_array = str.split('  ')
+  new_word_array.map { |word| new_str += "#{word.split.map { |text| decode_word(text) }.join} " }
+  new_str
```

Comment:
- [ ] **[OPTIONAL]** I think it would be a good idea to remove the whitespace at the end of the sentences.

  ```
  new_str.strip
  ```

  ![image](https://user-images.githubusercontent.com/81468675/217554547-a7b1a4d6-e24a-4555-a140-1aa9c9b80eeb.png)